vyper/parser/parser_utils.py

```diff
@@ -326,7 +327,7 @@ def add_variable_offset(parent, key, pos, array_bounds_check=True):
             return LLLnode.from_list(['sha3_64', parent, sub],
                                      typ=subtype,
                                      location='storage')
-        elif location == 'memory':
+        elif location in ('memory', 'calldata'):
```

Comment:
this could probably just be changed to a catch-all `else`?

Reply:
Yes, but I prefer keeping it future proof i.e. if some new exotic type is added and we handle it incorrectly.